tests/test_1294_to_and_from_parquet.py

```diff
@@ -8,6 +8,7 @@
 import awkward as ak
 
 pyarrow_parquet = pytest.importorskip("pyarrow.parquet")
+fsspec = pytest.importorskip("fsspec")
```

Comment:
`ffspec` must have recently become optional elsewhere; we didn't depend upon it explicitly in these tests, which started failing on Mac.

Reply:
`ak.to_parquet` and `ak.from_parquet` rely fairly heavily on it, something that I am reminded of when I make environment.yml files for tutorials. Perhaps the pathway ought to be cleared for the case when we have a path to a local file or want to pass a file-like object directly into `pyarrow.parquet.ParquetFile` or `ParquetWriter`, so that fsspec is only needed for remote files. That's an aside.

Reply:
My initial reaction is to make `fsspec` a required dependency for IO (not for the `awkward` package itself, but we could provide extras to include it), given how fundamental it is to other parts of the Python ecosystem. I recall there being pushback on that from collaboration maintainers, but you'll have a better recollection of that conversation than I will, I suspect!